src/EnergyPlus/HeatBalanceAirManager.cc

```diff
@@ -3462,7 +3462,7 @@ namespace HeatBalanceAirManager {
             if (Loop == 1)
                 gio::write(OutputFileInits, Format_721)
                     << "ZoneInfiltration"
-                    << "Design Volume Flow Rate {m3/s},Volume Flow Rate/Floor Area {m3/s/m2},Volume Flow Rate/Exterior Surface Area {m3/s/m2},ACH - "
+                    << "Design Volume Flow Rate {m3/s},Volume Flow Rate/Floor Area {m3/s-m2},Volume Flow Rate/Exterior Surface Area {m3/s-m2},ACH - "
```

Comment:
OK, so `m3/s-m2` is already the accepted unit conversion in E+, this output was simply entered incorrectly in this string.  :+1: